docs/src/components/home/GoldSponsors.tsx

```diff
@@ -61,6 +54,21 @@ const GOLDs = [
     description: 'The best place to buy Instagram followers & likes.',
     href: 'https://megafamous.com/?utm_source=MUI&utm_medium=referral&utm_content=homepage',
   },
+  {
+    src: 'https://images.opencollective.com/dialmycalls/f5ae9ab/avatar/256.png',
```

Comment:
The other image dimensions should be fixed too

```suggestion
    src: 'https://images.opencollective.com/dialmycalls/f5ae9ab/avatar/40.png',
```

Reply:
Where do I get these images with different dimensions hosted on Open Collective?

Reply:
I believe that the dimensions on OpenCollective and GitHub are lazily generated, you can set any value.